milvus/question_answering.py

```diff
@@ -0,0 +1,97 @@
+import os
```

Comment:
Now that you don't have length restrictions, maybe it would be good idea to bring some of the code comments from the notebook.

Reply:
I added a reference to the original example instead.